Avatar/Badge/Badge.css

```diff
@@ -0,0 +1,101 @@
+@import url(/Avatar/Avatar.css);
+body {
+    font-family: sans-serif;
+  }
+  .icon {
```

Comment:
I feel these indentation not necessary, and then everything looks good for me

Reply:
Resolved!

Reply:
Please approve